tests/test_display.py

```
@@ -0,0 +1,692 @@
+        """Wrong input should transition to game over state."""
+        game_screen.sequence = ['left', 'right']
+        game_screen.player_index = 0
+        game_screen.state = 'input'
+
+        game_screen._handle_input('up', 1000)  # Wrong button
+
+        assert game_screen.state == 'gameover'
+
+    def test_pause_stops_timer(self, game_screen, mock_pygame):
+        """Pausing should freeze the timer."""
+        mock_pg, _ = mock_pygame
```

Ruff (0.15.2)

[warning] Unpacked variable `mock_pg` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)